src/overlap_check/app/overlap_check.cpp

```diff
@@ -0,0 +1,74 @@
+
+#include "overlap.hpp"
+
+#include <set>
+#include <memory>
+#include "moab/Core.hpp"
+#include "moab/ProgOptions.hpp"
+
+#ifdef _OPENMP
+#include <omp.h>
+#endif
+
+using namespace moab;
+
+int main(int argc, char* argv[]) {
+
+  ProgOptions po("overlap_check: a tool that searches for overlaps in a DagMC geometry."
+                 "This is currently a non-exhaustive search.");
+
+  std::string filename;
+  int points_per_tri_edge {0};
+  int n_threads {0};
```

Comment:
This should be guarded as so to remove the unused variable for single-threaded builds:
```c++
#ifdef _OPENMP
  int n_threads {0};
#endif
```